<launch>
 <!-- Turtlesim Node-->
 
 <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

 <node respawn="true" pkg="ps3joy"
         type="ps3joy_node.py" name="PS3_turtle_joy" >
     <param name="dev" type="string" value="/dev/input/js0" />
     <param name="deadzone" value="0.12" />
   </node>
 




 <!-- Axes -->
 <param name="axis_linear" value="1" type="int"/> 
 <param name="axis_angular" value="0" type="int"/>
 <param name="scale_linear" value="1" type="double"/>
 <param name="scale_angular" value="1" type="double"/>
 <param name="deadman_button" value="11" type="double"/>

 <node pkg="bigbot" type="joyteleop" name="joyteleop"/>
 
</launch>
